repo/blob/gcs/gcs_internal_test.go

```diff
@@ -0,0 +1,69 @@
+// Package gcs implements Storage based on Google Cloud Storage bucket.
+package gcs
+
+import (
+	"context"
+	"io/fs"
+	"os"
+	"testing"
+
+	gcsclient "cloud.google.com/go/storage"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGCSStorageCredentialsHelpers(t *testing.T) {
+	ctx := context.Background()
+	scope := gcsclient.ScopeReadOnly
+	var fileMode fs.FileMode = 0644
+
```

Comment:
nit: it seems that these could be performed using subtests via `t.Run(...)`

Reply:
I was following the convention on other test file, but probably a good call to use `t.Run(...)`

Reply:
Fixed.